scripts/install

```diff
@@ -292,6 +342,16 @@ ynh_script_progression --message="Starting a systemd service..."
 ynh_systemd_action --service_name=$app-jicofo --action="start" --log_path="/var/log/$app/$app-jicofo.log"
 ynh_systemd_action --service_name=$app-videobridge --action="start" --log_path="/var/log/$app/$app-videobridge.log"
 
+if [ $(uname -m) == "armv7l" ] 
+then
+	# rm official jniwrapper to copy 
+	rm "$final_path/jitsi-videobridge/lib/jniwrapper-native-1.0-8-g04269a7.jar"
+	cp "/tmp/jitsi-install/jitsi-sctp/jniwrapper/native/target/jniwrapper-native-1.0-SNAPSHOT.jar" "$final_path/jitsi-videobridge/lib/"
```

Comment:
we can't add that stuff in the `INSTALL DEPENDENCIES` step or in a dedicated step of `SPECIFIC SETUP`

Reply:
i don't know exactly when jniwrapper-native is created, tried higher in the script but does not work.
sure i have to investigate a bit more